023c20b

commit 023c20b
Author: Alain Delree <alain.delree@gmail.com>
Date:   Tue Aug 11 13:56:04 2026 +0200

    Corrige chemin ActualiseUI onefile dans rebuild_actualise_setup.bat et actualise.iss (issue #47)

diff --git a/build/rebuild_actualise_setup.bat b/build/rebuild_actualise_setup.bat
index 67fa8fc..827a01a 100644
--- a/build/rebuild_actualise_setup.bat
+++ b/build/rebuild_actualise_setup.bat
@@ -127,14 +127,14 @@ if not exist "dist\Actualise\Actualise.exe" (
     popd
     exit /b 1
 )
-if not exist "dist\ActualiseUI\ActualiseUI.exe" (
+if not exist "dist\ActualiseUI.exe" (
     echo.
-    echo ERREUR : ActualiseUI.exe introuvable dans dist\ActualiseUI apres le build.
+    echo ERREUR : ActualiseUI.exe introuvable dans dist apres le build.
     popd
     popd
     exit /b 1
 )
-echo dist\Actualise\Actualise.exe et dist\ActualiseUI\ActualiseUI.exe presents. OK.
+echo dist\Actualise\Actualise.exe et dist\ActualiseUI.exe presents. OK.
 echo.
 
 REM --- 7. Compiler l'installeur InnoSetup ------------------------------------
diff --git a/installeur/actualise.iss b/installeur/actualise.iss
index cb093f7..f6fc3a6 100644
--- a/installeur/actualise.iss
+++ b/installeur/actualise.iss
@@ -24,7 +24,7 @@
 ; Scrabble/Rummikub) : PyInstaller/ISCC ne doivent jamais tourner
 ; directement sur le partage VirtualBox (\\VBOXSVR\...).
 #define ActualiseSourceDir "C:\Temp\ActualiseBuild\dist\Actualise"
-#define ActualiseUISourceDir "C:\Temp\ActualiseBuild\dist\ActualiseUI"
+#define ActualiseUISourceDir "C:\Temp\ActualiseBuild\dist"
 
 [Setup]
 AppId={{8556F922-AD4A-4E8D-A7E9-08AD815D5552}
